src/components/pages/course-view/CourseViewContainer.jsx

```diff
@@ -1,56 +1,50 @@
 import React, { useState } from 'react';
 import { Card } from 'antd';
+import { ModuleView } from '../index';
 import styled from 'styled-components';
 
+//Component Styles
```

Comment:
Question: Should we find a set pattern for using Styled-Components and where we should put them? I'm comfortable with hunting if you'd rather not enforce a rule on this but it could end up being nice to have a `CourseViewContainer.styled.jsx` file (for example) for every component that uses styled and import them in?

I don't really care a lot, esp. cuz our SComponents are going to be really small with AntD but just thinking out loud here.

Reply:
We should totally discuss this as a team. I do like the idea of pulling out styles to a separate file that is imported in. It'll keep the code more legible, especially as these styles grow to fit the needs of the stakeholder. I think it's a good idea, but let's circle back and make sure the whole team is onboard!